john-johnconlon/Homework-1/99-bottles-of-beer/app.js

```diff
@@ -12,9 +12,14 @@
 
 for (let beerIndex = 99; beerIndex >= 0; beerIndex--) {
 	if (beerIndex >= 1) {
-  console.log(beerIndex + " bottles of beer on the wall " + beerIndex + " bottles of beer." + "\n" + "take one down, pass it around " + (beerIndex -1) + " bottles of beer on the wall");
+  listItem = document.createElement('li');
```

Comment:
It's fantastic to see you came back to refactor your first assignment, John! Keep that up as you learn and you'll grow quickly as a skilled developer.